models/web/snowplow_media_pivot_base.sql

```diff
@@ -0,0 +1,42 @@
+{{
+  config(
+    materialized='table'
```

Comment:
There's no need to define the materialisation as `table` here when you have that in the `dbt_project.yml` file already

Reply:
I`ll be changing it to view just to be in line with the web model from this perspective, too.